tests/unit/test-cmd/test.run.js

```diff
@@ -329,6 +329,26 @@ describe('run', () => {
     );
   });
 
+  it('provides a chromiumDebuggerPort option to the Chromium runner', async () => {
```

Comment:
This test is not enough for test coverage. It merely verifies that the CLI parameter is passed to the extension runner.

You need to add more tests, to verify that the port is received correctly in https://github.com/mozilla/web-ext/blob/005bed86dff796cca48281b1928f7109b0972c80/tests/unit/test-extension-runners/test.chromium.js . And the check requested in the other comment (i.e. validity of port) should be tested too.